.github/workflows/selt_test.yml

```diff
@@ -0,0 +1,14 @@
+jobs:
```

Comment:
`.github/workflows/selt_test.yml` - self, not selt?

Reply:
oops, Fixed in https://github.com/w3c/trace-context/pull/455/commits/589c051800651927f8efdfddff48f148df899551